spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherRunner.java

```diff
@@ -58,7 +58,7 @@ public void run(String... args) throws Exception {
 			}
 			log.info(sb.toString());
 		}
-		this.moduleLauncher.launch(launchRequests);
+		this.moduleLauncher.launch(launchRequests, moduleLauncherProperties.isAggregate(), filterSpringProperties(args));
```

Comment:
I think this is up for debate. What Ilaya has pointed out was that if we pass all arguments to the parent context, then, due to the hierarchical nature of environments they may become visible to the children. `spring.*` props can help with configuring the binding, which is performed at the aggregate (parent) level.